prow/test/integration/integration-test.sh

```diff
@@ -39,4 +39,6 @@ main() {
   "${bazel}" test //prow/test/integration/test:go_default_test --action_env=KUBECONFIG=${HOME}/.kube/config --test_arg=--run-integration-test "$@" || ( echo "FAILED: running tests">&2; return 1 )
 }
 
-main "$@"
+time bazel test --config=ci --nobuild_tests_only //...
```

Comment:
Please include it with L39 so things run in parallel

Reply:
Test flag doesn't work, I can switch it to use env var

Reply:
Another option would be to put the arg into the sh_test or whatever target